### PR TITLE
Fix wrong @return annotations

### PR DESCRIPTION
**Pull request by MatthiasKuehneEllerhold**
Just a quick fix for two wrong @return annotations. (Well and my IDE removed the trailing spaces...)

### CODE REVIEW

File: src/MimePart.php [hunk 1]
```diff
@@ -464,15 +464,15 @@ private function copyContentStream($fromHandle, $toHandle, $isUUEncoded)
         rewind($fromHandle);
         // changed from stream_copy_to_stream because hhvm seems to stop before
         // end of file for some reason
-        while (($read = fread($fromHandle, 1024)) != false) {
+        while (($read = fread($fromHandle, 1024)) !== false) {
```
Comment (zbateson) [inline src/MimePart.php]: @MatthiasKuehneEllerhold 

Travis is stalling.  Probably from this change :smile:

**Comment by zbateson** [comment]
Hi @MatthiasKuehneEllerhold 

Awesome, thanks for the contribution!

It seems that travis is failing with this change.  I've commented on the line (probably) responsible... I didn't notice any other code changes.

**Comment by MatthiasKuehneEllerhold** [comment]
I've reverted the change in this line as per your request.

**Comment by zbateson** [comment]
Excellent, thank you

**Pull request merged**